base/range.jl

```diff
@@ -41,6 +41,8 @@ equal to 1, which produces:
 `a:s:b` is similar but uses a step size of `s` (a [`StepRange`](@ref) or
 [`StepRangeLen`](@ref)). See also [`range`](@ref) for more control.
 
+To create a reverse range, use `reverse(a:b)` or a negative step size, e.g. `b:-1:a`.
```

Comment:
```suggestion
To create a reverse-order range, use `reverse(a:b)` or a negative step size, e.g. `b:-1:a`.
```

Reply:
Or perhaps 'to create a decreasing range'?

Reply:
Maybe "descending range" would be more precise.

(After all, `0:-1:-10` is descending algebraically, but is increasing in magnitude.)